docs/web-component.md

```diff
@@ -20,9 +20,9 @@ meta -->
 
 ### Overview
 
-Luigi offers the possibility to open a micro frontend as a web component; if you want to have more information about web component, please have a look at the page: [Web Component](https://developer.mozilla.org/en-US/docs/Web/Web_Components).
+Luigi offers the possibility to open a micro frontend as a web component; if you want to have more information about web components, please have a look at the page: [Web Component](https://developer.mozilla.org/en-US/docs/Web/Web_Components).
 
-For no-complex micro frontend, it can be a good and fast alternative: all the frontend will be loaded in a single javascript file.
+For non-complex micro frontend, it can be a good and fast alternative: all the frontend will be loaded in a single javascript file.
```

Comment:
"all the frontend" sounds a bit strange to me, and it is not relevant here imho.
you should rather mention, that it is for trusted sources only.

Reply:
Hi, I'm not sure why this change is here in the blog post PR, I intended to open a new PR with a review of the web components doc. So I will try to remove this and add all additional changes in a new PR.